//package Day8;
//import java.util.Scanner;
//
//public class ExceptionExam02 {
//	public static void main(String[] args) throws ArithmeticException,InterruptedException{
//
//		Scanner sc = new Scanner(System.in);
//		int num = sc.nextInt();
//		System.out.println(3 / num);
//
////		try {
//			Thread.sleep(3000);
////		} catch (InterruptedException e) {
////			// TODO Auto-generated catch block
////			e.printStackTrace();
////		}
//		
//		System.out.println("Bye");
//	}
//}